torch/quantization/fx/quantization_patterns.py

```diff
@@ -814,11 +818,15 @@ def convert(self, quantizer: QuantizerCls, node: Node, load_arg: Callable,
             torch.nn.InstanceNorm3d: int8_dtypes,
             torch.nn.LayerNorm: all_dtypes,
             torch.nn.SiLU: fp16_dtypes,
+            torch.nn.GELU: all_dtypes,
+            torch.nn.Softmax: all_dtypes,
```

Comment:
should this be `fp16_dtypes`?

Reply:
are we adding this for int8 or fp16? I saw int8 in the docs

Reply:
I thoguht the goal was for int8 compatibility. From the design doc:

"The way we can implement this in FX graph mode quantization is to add a "reference pattern" for gelu, which mimics gelu_int8 without an actual gelu_int8 kernel."

Reply:
@HDCharles , you're right.  It should be `int8` for the current request, I misspoke earlier.  `all_dtypes` sounds good.